source/_integrations/drop_connect.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: DROP
+description: Instructions on how to integrate DROP into Home Assistant.
+ha_category:
+  - Sensor
+ha_config_flow: true
+ha_codeowners:
+  - '@ChandlerSystems'
+  - '@pfrazer'
+ha_domain: drop_connect
+ha_iot_class: Local Push
+ha_mqtt: true
+ha_release: '2024.1'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The DROP integration provides connectivity with
+[DROP smart water system](https://dropconnect.com) devices via a local MQTT API.
+
+### Supported devices
+
+There is currently support for the following DROP products within Home Assistant:
+
+- **DROP Hub**: centralized system status and control.
+- **Leak Detector**: reports detected leaks around the home.
+- **Softener**: improves water quality by decreasing water hardness with integral water shutoff.
+- **Filter**: improves water quality by removing contaminants with integral water shutoff.
+- **Protection Valve**: automatic water shutoff with slow leak detection.
+- **Pump Controller**: smart replacement for an FSG pressure switch.
+- **RO Filter**: reverse osmosis drinking water filtration.
+- **Salt Sensor**: alerts when the salt level in the softener brine tank is low.
+
+### Configuration
+
+To use DROP with Home Assistant, you must have already configured the [MQTT](/integrations/mqtt/) platform. It is
+recommended
+that you create a unique user to connect your DROP Hub to your MQTT broker. If you are
+using the Mosquitto add-on, go to **Settings > Add-ons** in the Home Assistant UI and select the Mosquitto broker.
+Under **Configuration**, enter a new username and password for your DROP Hub in the **Logins** field:
+
+```yaml
+- username: drop
+  password: your_unique_password
+```
+
+If you are using a different MQTT broker, please reference its documentation to learn how to add a new user.
+
+To enable the DROP integration, use the DROP Connect app to connect your DROP Hub to your MQTT broker:
```

Comment:
We have many users not using add-ons... this is super confusing as this is integration documentation and not device or add-on documentation

Reply:
I was the one who suggested this, I'll open a followup PR on next.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/30527